data/xp-pen-ack05-remote.tablet

```diff
@@ -0,0 +1,35 @@
+# XP-Pen
+# ACK05 Remote
+#
+# Reports as UGTABLET Artist Pro 16 (Gen2), for recordings
+# etc. see https://gitlab.freedesktop.org/libevdev/udev-hid-bpf/-/issues/32
+#
+#  ┌────────┐────────────────────────┐
+# /  ┌────┐  \   ┌───┐┌───┐┌───┐┌───┐│
+# │  │Dial│  │   │ A ││ B ││ C ││   ││
+# │  └────┘  │   └───┘└───┘└───┘│ G ││
+# │\________/    ┌───┐┌───┐┌───┐│   ││
+# │              │ D ││ E ││ F ││   ││
+# │              └───┘└───┘└───┘└───┘│
+# │              ┌───┐┌────────┐┌───┐│
+# │              │ H ││    I   ││ J ││
+# │              └───┘└────────┘└───┘│
+# └──────────────────────────────────┘
+
+[Device]
+Name=UGTABLET Artist Pro 16 (Gen2)
+ModelName=
+DeviceMatch=usb|28bd|095b
+Layout=xp-pen-ack05-remote.svg
+Class=Remote
+
+[Features]
+Stylus=false
+NumDials=1
+NumRings=0
+NumStrips=0
+StatusLEDs=Ring
+
+[Buttons]
+Left=A;B;C;D;E;F;G;H;I;J;
+EvdevCodes=BTN_0;BTN_1;BTN_2;BTN_3;BTN_4;BTN_5;BTN_6;BTN_7;BTN_8
```

Comment:
Maybe also a BTN_10 : the keypad part has 10 buttons  (so 0 to 9) but the Dial has a middle button too.

I saw on the Wacom EKR this one (middle dial button) is counted as BTN_0 and "A" label. 

So, I'll probably do that for the SVG (correct me if I'm wrong.)

Reply:
> BTN_9 seems to be missing :)

Counting is not my forte... fixed, thanks.

> Maybe also a BTN_10 : the keypad part has 10 buttons (so 0 to 9) but the Dial has a middle button too.

does the dial button send BTN_0? If it does we shift all of them up and name the dial button A.

Reply:
On my side I managed to get teh dial button sending something sensible (and using currently `BTN_LEFT` because `BTN_10` doesn't exist, opened to discussion).

On the [official documentation](https://www.xp-pen.com/user-manual/ack05-wireless-shortcut-remote) of the product, it is said that the button on the dial is intended for "mode switching".

I think we should have something like that (completely untested):

```
[Features]
Stylus=false
NumRings=1
NumDials=1
NumStrips=0

[Buttons]
Left=A;B;C;D;E;F;G;H;I;J;K
EvdevCodes=BTN_0;BTN_1;BTN_2;BTN_3;BTN_4;BTN_5;BTN_6;BTN_7;BTN_8;BTN_9;BTN_LEFT;

#Note that no LEDs light up on the device itself
RingNumModes=4
Ring=K
```